jack's low API version (so:libjack.so.0.1.0) makes apk prefer pipewire-jack over jack
We don't want this, users need to explicitly tell users they prefer pipewire-jack, so we increase the API version here to be always higher than pipewire-jack

diff --git a/wscript b/wscript
index e0d32c8d..2c0a7123 100644
--- a/wscript
+++ b/wscript
@@ -13,7 +13,7 @@ from waflib.Build import BuildContext, CleanContext, InstallContext, UninstallCo
 
 VERSION='1.9.20'
 APPNAME='jack'
-JACK_API_VERSION = '0.1.0'
+JACK_API_VERSION = '0.999.0'
 
 # these variables are mandatory ('/' are converted automatically)
 top = '.'
